ouroboros-consensus/src/Ouroboros/Consensus/Protocol/WithEBBs.hs

```diff
@@ -1,3 +1,6 @@
+{-# LANGUAGE BangPatterns            #-}
+{-# LANGUAGE DeriveAnyClass          #-}
```

Comment:
Why is this needed? Or `DeriveGeneric`?

Reply:
It's not yet, but it will in my next PR :) Remnant from when I rebased this on `master`. But we'll need it as soon as I submit the next (big) PR.